corehq/apps/email/migrations/0002_auto_20230918_0436.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 3.2.20 on 2023-09-18 04:36
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
```

Comment:
can these 3 migrations be merged into one?

Reply:
Merged